testing/openjdk12/Alpine_Bug_10126.java

```diff
@@ -0,0 +1,13 @@
+public class Alpine_Bug_10126 {
+    public static void main(String[] args) throws Exception {
+        try (java.net.Socket sock = javax.net.ssl.SSLSocketFactory.getDefault().createSocket("bugs.alpinelinux.org", 443);
```

Comment:
Connecting to bugs is not the best idea now.

Reply:
Thanks, updated